### DISCUSSION TITLE
File icons

### DISCUSSION THREAD

**Discussion by kema-dev** [community/community#43870] [Repositories]
### Select Topic Area

Product Feedback

### Body

:wave:
File icons are now very common, but I'm surprised GitHub does not support it. I personally use [an extension](https://github.com/Claudiohbsantos/github-material-icons-extension) to beautify the UI, but I think that a native support would be great.
Below is an image of the UI with extension enabled (image from extension's repo)
![image](https://user-images.githubusercontent.com/54537427/211600137-75795caa-e634-4c67-91d8-b91692f33cd7.png)

**Comment by spenserblack** [comment]
I'm surprised this request isn't more popular! When looking through a long list of files, icons help me quickly find the file I'm looking for.
